src/python/pants/backend/python/targets/python_target.py

```diff
@@ -1,8 +1,10 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
```

Comment:
We had to update the V1 definition of `PythonTarget` so that V1 code does not fail when encountering `constraints` in a Python target.